src/ontology/mondo-edit.obo

```diff
@@ -152626,24 +152612,34 @@ relationship: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/39
 
 [Term]
 id: MONDO:0008319
-name: protoporphyria, erythropoietic
+name: erythropoietic protoporphyria
 def: "A rare congenital metabolic disorder characterized by an inborn error of porphyrin-heme biosynthesis. It is caused by deficiency of the enzyme ferrochelatase. Signs and symptoms include painful cutaneous photosensitivity leading to blistering and scarring of the exposed skin areas, erythrodontia, red discoloration of urine, hemolytic anemia, and splenomegaly." [NCIT:C84698]
 subset: gard_rare
-synonym: "EPP" RELATED [GARD:0004527, MONDO:Lexical, OMIM:177000]
-synonym: "Erythrohepatic protoporphyria" RELATED [GARD:0004527, OMIM:177000]
-synonym: "erythropoietic protoporphyria" EXACT [NCIT:C84698]
-synonym: "erythropoietic protoporphyria" RELATED [NCIT:C84698]
+synonym: "EPP" EXACT EXCLUDE [DOID:13270]
+synonym: "EPP" RELATED ABBREVIATION [GARD:0004527, MONDO:Lexical, OMIM:177000]
+synonym: "EPP (erythropoietic protoporphyria porphyria)" EXACT [CSP2005:1849-7560, DOID:13270]
+synonym: "erythrohepatic protoporphyria" RELATED [GARD:0004527, NCIT:C84698, OMIM:177000]
+synonym: "erythropoietic protoporphyria" EXACT [MONDO:0001676, NCIT:C84698]
 synonym: "ferrochelatase deficiency" RELATED [GARD:0004527, OMIM:177000]
 synonym: "heme synthetase deficiency" RELATED [GARD:0004527, OMIM:177000]
+synonym: "protoporphyria" EXACT [DOID:13270, MTHICD9_2006:277.1]
 synonym: "protoporphyria, erythropoietic" EXACT [MONDO:Lexical, OMIM:177000]
 synonym: "protoporphyria, erythropoietic; EPP" RELATED [OMIM:177000]
+xref: DOID:13270 {source="MONDO:equivalentTo"}
 xref: GARD:0004527 {source="MONDO:equivalentTo"}
-xref: NCIT:C84698 {source="MONDO:equivalentTo", source="MONDO:kboom-pr-0.90/0.76/0.36"}
-xref: OMIM:177000 {source="GARD:0004527", source="MONDO:equivalentTo"}
+xref: ICD10:E80.0 {source="DOID:13270"}
+xref: MESH:D046351 {source="DOID:13270", source="MONDO:equivalentTo"}
+xref: NCIT:C84698 {source="DOID:13270", source="MONDO:equivalentTo", source="MONDO:kboom-pr-0.90/0.76/0.36"}
+xref: OMIM:177000 {source="DOID:13270", source="GARD:0004527", source="MONDO:equivalentTo"}
```

Comment:
We should never be equiv to a leaf OMIM and an OMIMPS - can you add a ticket for a check for this?

See https://omim.org/entry/177000

Reply:
I created a ticket